src/libraries/System.Linq/src/System/Linq/Take.SpeedOpt.cs

```diff
@@ -12,35 +11,5 @@ public static partial class Enumerable
             source is IPartition<TSource> partition ? partition.Take(count) :
             source is IList<TSource> sourceList ? (IEnumerable<TSource>)new ListPartition<TSource>(sourceList, 0, count - 1) :
             new EnumerablePartition<TSource>(source, 0, count - 1);
-
-        private static IEnumerable<TSource> TakeLastEnumerableFactory<TSource>(IEnumerable<TSource> source, int count)
```

Comment:
Delete it from the `Task.SizeOpt.cs` as well?

Reply:
Yup